common/menu-entry.c

```diff
@@ -293,6 +294,30 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
         /*if (shortcut)
             shortcutFree(&sc);*/
     }
+    if (me->type == ENTRY_TYPE_THEME) {
+        config_t cfg = {0};
+        config_init(&cfg);
+        config_setting_t *themeInfo;
+        const char *name,
+                   *author = textGetString(StrId_DefaultPublisher),
+                   *version = "1.0.0";
+                   
+        if(config_read_file(&cfg, me->path)) {
+            themeInfo = config_lookup(&cfg, "themeInfo");
+            if (themeInfo != NULL) {
+                if(config_setting_lookup_string(themeInfo, "name", &name))
+                    strncpy(me->name, name, sizeof(me->name)-1);
+                if(!config_setting_lookup_string(themeInfo, "author", &author))
+                    author = textGetString(StrId_DefaultPublisher);
+                if(!config_setting_lookup_string(themeInfo, "version", &version))\
+                    version = "1.0.0";
```

Comment:
redundant

Reply:
so author is not overwritten with null if it the config_setting_lookup_string does not find a string?

Reply:
https://hyperrealm.github.io/libconfig/libconfig_manual.html#index-config_005fsetting_005fget_005fstring

Reply:
fixed